Quality-Assistance.md

```diff
@@ -2,34 +2,95 @@
+- I am learning how to help product teams make pragmatic decisions during the fixed timelines of our build cycles, taking the bigger picture into account.
+- I am learning how to organise, coordinate, and run quality coaching activities, like QA KickOff and QA Demo, and how to reap the most value from these activities.
+
+# Senior Quality Coach (Level 3)
+
+> Building on the Quality Coach (Level 2) role, I am a proficient Quality Coach, helping Octopus product delivery teams and their members to deliver work with increasing quality over time with the ultimate goal of these teams becoming self-sufficient.
+
+## I understand our customers and our business
+
+- I have a good general understanding of the problems our customers are trying to solve and how we do/don't solve them.
+- I have attained mastery in several customer problem+solution areas, and mastering more is just a matter of time and opportunity.
+- I can comfortably set up a build and delivery pipeline for a real-life application.
+- I am actively building good relationships with people in other teams like support, customer advisory, trust and security, and customer success.
+
+## I am an effective and efficient quality coach
+
+- I am well-connected to our product delivery teams, including product managers, product designers, and software engineers.
+- I can comfortably and effectively coach multiple teams in the same cycle, perhaps across different parts of the business domain.
+- I confidently help teams increase the flow of value to our customers, where work flows forward and rarely comes back due to quality concerns we missed on the first pass.
+- I confidently partner with teams using an iterative development approach, from walking skeleton to polished product.
+- I confidently partner with teams through building smaller improvements within a single cycle, or complex value streams spanned across multiple cycles.
+- I confidently help product teams make pragmatic decisions during the fixed timelines of our build cycles, taking the bigger picture into account.
+- I confidently organise, coordinate, and run quality coaching activities, like QA KickOff and QA Demo, and how to reap the most value from these activities.
+
+# Lead Quality Coach (Level 4)
+
+> Building on Senior Quality Coach (Level 3), I am a force multiplier, deliberately shifting my focus from individual quality coaching toward uplifting the whole product delivery team.
+
+## I am an expert in our customers and our business and could be an effective product manager or customer advocate
+
+- I have a broad and thorough understanding of the problems our customers are trying to solve and how we do/don't solve them, helping me make good tradeoffs with the entire picture in mind.
+- In addition to our product delivery teams, I am well-connected to our other teams, like support, customer advisory, trust and security, and customer success.
```

Comment:
Isn't `Customer success team` now `Customer advisory team`?

Reply:
Yes, and `Sales` is `Customer Success`, I think.